settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
I'm no Gradle expert, but could the break be caused by the gradle 2.3 in the `analyzer` directory? I have tried to update it to 2.10 and the build succeeds if launched from the `analyzer` folder.

Also, if we keep `analyzer` in `settings.gradle` we can remove the `analyzer/gradle` folder and gradlew files, right?